docs/DeveloperGuide.md

```diff
@@ -703,39 +700,198 @@ testers are expected to do more *exploratory* testing.
 
 </div>
 
-### Launch and shutdown
+### Launch and Shutdown
 
-1. Initial launch
+1. Initial Launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Double-click the jar file.<br>
+      Expected: GUI runs with a set of sample modules. The window size may not be optimum.
 
-1. Saving window preferences
+1. Saving Window Preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+1. Shutdown
+
+   1. Test case: Click the "close" button (red button) at the top of the window.<br>
+      Expected: The window closes immediately.
+      
+   1. Test case: `exit`<br>
+      Expected: GUI shows a farewell message,"Exiting GradPad as requested ..." and delays for 1.5 seconds, after which the window closes.
```

Comment:
Do we currently have the 1.5 secs time delay implemented?

Reply:
not yet but im planning to implement it!

Reply:
Okay dope